### ISSUE TITLE
只用一张图片训练为什么val loss很高，trainloss很低？

### ISSUE THREAD

**Issue by northeastsquare** [xuannianz/EfficientDet#175]
多边形检测，不做random transform, 手工查看train时候图片的输入尺寸和val时候的图片尺寸是一样的，为什么train loss 可以到小数点后4位，
1/2 [==============>...............] - ETA: 0s - loss: 0.0019 - classification_loss: 3.9244e-04 - regression_loss: 0.0015Epoch 1/10000

valloss在0.9-0.4左右
Epoch 00063: val_loss improved from 0.76781 to 0.76651, saving model to checkpoints/2020-05-24/csv_63_0.0016_0.7665.h5

而用保存的权值文件inference, 肉眼看，准确率不高

**Comment by northeastsquare** [comment]
1/2 [==============>...............] - ETA: 0s - loss: 8.9618e-05 - classification_loss: 2.2625e-05 - regression_loss: 6.6992e-05Epoch 1/10000

Epoch 00638: val_loss improved from 0.54733 to 0.54349, saving model to checkpoints/2020-05-24/csv_638_0.0003_0.5435.h5
